spec/factories/courses.rb

```diff
@@ -1,4 +1,12 @@
 FactoryGirl.define do
+  factory :user do 
+    first_name "John"
+     last_name "Smith"
+     email "js@email.com"
+     # course_id course.id
```

Comment:
We are using it in course_spec.rb
`  it "has many users" do
    course = FactoryGirl.build(:course)
    john = FactoryGirl.build(:user)
    tom = FactoryGirl.build(:user, first_name: "tom")
    course = FactoryGirl.create(:course)
    
    course.users << john << tom 
    expect(course.users[0].id).to eq(john.id)
  end`